.github/workflows/test.yml

```
@@ -50,9 +50,9 @@ jobs:
       - run: sam init --name sam-app --runtime nodejs14.x --dependency-manager npm --app-template hello-world
 
       # Test sam build
-      - uses: actions/setup-node@v1
+      - uses: actions/setup-node@v3
         with:
-          node-version: "14"
+          node-version: "16"
```

This should be the same version as used during `sam init`. SAM CLI doesn't yet support Node.js 16, so we can keep it as-is for now:

```suggestion
          node-version: "14"
```